src/test/java/org/zalando/jackson/datatype/money/MonetaryAmountSerializerTest.java

```diff
@@ -28,19 +28,23 @@
 
     private final MonetaryAmount amount;
     private final MonetaryAmount hundred;
+    private final MonetaryAmount fractions;
 
-    public MonetaryAmountSerializerTest(final MonetaryAmount amount, final MonetaryAmount hundred) {
+    public MonetaryAmountSerializerTest(final MonetaryAmount amount, final MonetaryAmount hundred,
+            final MonetaryAmount fractions) {
         this.amount = amount;
         this.hundred = hundred;
+        this.fractions = fractions;
     }
 
     @Parameters(name = "{0}, {1}")
     public static Iterable<Object[]> data() {
         return Arrays.asList(new Object[][]{
-                {FastMoney.of(29.95, "EUR"), FastMoney.of(100.00, "EUR")},
-                {Money.of(29.95, "EUR"), Money.of(100.00, "EUR")},
+                {FastMoney.of(29.95, "EUR"), FastMoney.of(100.00, "EUR"), FastMoney.of(0.0001, "EUR")},
```

Comment:
The test case for expanding to default number of digits may be missing when using this inputs. We should use `100` instead of `100.00`.

Reply:
True